### PR TITLE
Add test for interaction between defaultEndpointToHostAction and wildcard HEPs

### PR DESCRIPTION
**Pull request by fasaxc**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

File: felix/fv/ep_to_host_action_test.go [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+			cc.Expect(connectivity.Some, w[0], w[1])
+			cc.CheckConnectivity()
+		},
+		entry("DROP", "Accept", "none", "", connectivity.Some),
+		entry("DROP", "Return", "none", "", connectivity.None),
+		entry("DROP", "Drop", "none", "", connectivity.None),
+
+		entry("ACCEPT", "Accept", "none", "", connectivity.Some),
+		entry("ACCEPT", "Return", "none", "", connectivity.Some),
+		entry("ACCEPT", "Drop", "none", "", connectivity.None),
```
Comment (tomastigera) [inline felix/fv/ep_to_host_action_test.go:208]: Is this true? I'd expect that the remote pod would connect in this case.

File: felix/fv/ep_to_host_action_test.go [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+				f.TriggerDelayedStart()
+			}
+
+			// Now add the default allow profile, which should give us WEP-to-WEP connectivity.
+			// When we get WEp-to-WEP, we know that Felix has finished programming so the
+			// WEP-to-host test is valid.
+
+			By("Checking connectivity")
+			cc.Expect(expectedConn, w[0], hostW[0])
+			cc.Expect(expectedConn, w[1], hostW[1])
```
Comment (tomastigera) [inline felix/fv/ep_to_host_action_test.go:198]: I'd expect local and remote pod to have possibly different connectivity. If the epToHost == DROP, the remote pod should still connect if other policies permit, right?

Comment (fasaxc) [reply felix/fv/ep_to_host_action_test.go:198]: They're each connecting to their own host, so should be symmetric?

**Comment by tomastigera** [comment]
LGTM, included in https://github.com/projectcalico/calico/pull/8388

**Pull request closed without merging**